### PR TITLE
Stuck with Intersection on large volume data

### PR DESCRIPTION
**Pull request by stone-wlg**
Issue:
- Guest (10,000,000 ID SM3)
image
- Host (10,000,000 ID SM3)
image

Env:
- Fate 1.7.2
- VM: 64C / 64G / 1TB * 3

Solution:
- add more resource for pulsar
- adjust pulsar config as pull requests

Result:
image

### CODE REVIEW

**Comment by owlet42** [comment]
@stone-wlg Thanks for the contribution. I have a few questions:
How is pulsar deployed and how can I add more resources to pulsar?

File: helm-charts/FATE/templates/pulsar.yaml [hunk 1]
```diff
@@ -134,7 +134,7 @@ data:
     brokerDeleteInactiveTopicsEnabled=true
     
     # How often to check for inactive topics
-    brokerDeleteInactiveTopicsFrequencySeconds=60
+    brokerDeleteInactiveTopicsFrequencySeconds=36000
```
Comment (JingChen23) [inline helm-charts/FATE/templates/pulsar.yaml:137]: My understanding is, 60 is too frequent so will eat too much resource, is that right?

File: helm-charts/FATE/templates/pulsar.yaml [hunk 2]
```diff
@@ -125,7 +125,7 @@ data:
     backlogQuotaCheckIntervalInSeconds=60
     
     # Default per-topic backlog quota limit
-    backlogQuotaDefaultLimitGB=10
+    backlogQuotaDefaultLimitGB=-1
```
Comment (JingChen23) [inline helm-charts/FATE/templates/pulsar.yaml:128]: What is the effect of this change?

Comment (LaynePeng) [reply helm-charts/FATE/templates/pulsar.yaml:128]: Unlimit the backlog quota may cause the storage not enough problem. Suggest pass through this setting to user, and set -1 as the default value.

Comment (stone-wlg) [reply helm-charts/FATE/templates/pulsar.yaml:128]: Yes, unlimited backlog quota, but  brokerDeleteInactiveTopicsFrequencySeconds will delete it when pulsar recognize the topic as inactive.

**Comment by stone-wlg** [comment]
> @stone-wlg Thanks for the contribution. I have a few questions:
> How is pulsar deployed and how can I add more resources to pulsar?

I take it out of from k8s, but it is still standalone.
try it add more resource for pulsar.
environment:
PULSAR_MEM: "-Xms16g -Xmx16g -XX:MaxDirectMemorySize=16g"

File: helm-charts/FATE/templates/pulsar.yaml (hunk 2 above)
Comment (LaynePeng) [reply helm-charts/FATE/templates/pulsar.yaml:128]: I am not sure if I understand it right, but the space needed for the backlog = messages number * message size * delete frequency. So if the storage is less than accumulated messages in 36000 seconds (we set in default), we will still run out of storage. Right?

**Pull request merged**